src/DssSpellManager.t.sol

```diff
@@ -0,0 +1,239 @@
+}
+
+interface MemberList {
+    function updateMember(address, uint) external;
+}
+
+interface AssessorLike {
+    function calcSeniorTokenPrice() external returns (uint);
+}
+
+interface FileLike {
+    function file(bytes32 what, address data) external;
+}
+
+interface ERC20Like {
+    function mint(address, uint256) external;
+}
+
+interface TinlakeManagerLike {
+    function gem() external view returns (address);
+    function wards(address) external view returns (uint);
+    function lock(uint256 wad) external;
+    function join(uint256 wad) external;
+    function draw(uint256 wad) external;
+    function wipe(uint256 wad) external;
+    function exit(uint256 wad) external;
+}
+
+contract DssSpellManagerTest is DSTest, DSMath {
+
+    Hevm hevm;
+    DssSpell spell;
+    Addresses addr  = new Addresses();
+
+    DSChiefAbstract chief = DSChiefAbstract(addr.addr("MCD_ADM"));
+    DaiAbstract dai = DaiAbstract(addr.addr("MCD_DAI"));
+    DSTokenAbstract gov = DSTokenAbstract(addr.addr("MCD_GOV"));
+
+    uint constant initialSpellDaiBalance = 10000 ether;
+    uint constant initialSpellDropBalance = 1000 ether;
+
+    CentrifugeCollateralTestValues[] collaterals;
+
+    function setUp() public {
+        hevm = Hevm(HEVM_ADDRESS);
+        spell = new DssSpell();
+
+        collaterals.push(CentrifugeCollateralTestValues({
+            ilk: "RWA003",
+            LIQ: 0x2881c5dF65A8D81e38f7636122aFb456514804CC,
+            URN: 0x7bF825718e7C388c3be16CFe9982539A7455540F,
+            ROOT: 0xdB3bC9fB1893222d266762e9fF857EB74D75c7D6,
+            COORDINATOR: 0xFc224d40Eb9c40c85c71efa773Ce24f8C95aAbAb,
+            DROP: 0x5b2F0521875B188C0afc925B1598e1FF246F9306,
+            MEMBERLIST: 0x26129802A858F3C28553f793E1008b8338e6aEd2,
+            MGR: 0x2A9798c6F165B6D60Cfb923Fe5BFD6f338695D9B
+        }));
+
+        collaterals.push(CentrifugeCollateralTestValues({
+            ilk: "RWA004",
+            LIQ: 0x2881c5dF65A8D81e38f7636122aFb456514804CC,
+            URN: 0xeF1699548717aa4Cf47aD738316280b56814C821,
+            ROOT: 0x4cA805cE8EcE2E63FfC1F9f8F2731D3F48DF89Df,
+            COORDINATOR: 0xE2a04a4d4Df350a752ADA79616D7f588C1A195cF,
+            DROP: 0xd511397f79b112638ee3B6902F7B53A0A23386C4,
+            MEMBERLIST: 0x1Bc55bcAf89f514CE5a8336bEC7429a99e804910,
+            MGR: 0xe1ed3F588A98bF8a3744f4BF74Fd8540e81AdE3f
+        }));
+
+        collaterals.push(CentrifugeCollateralTestValues({
+            ilk: "RWA005",
+            LIQ: 0x5b702e1fEF3F556cbe219eE697D7f170A236cc66,
+            URN: 0xc40907545C57dB30F01a1c2acB242C7c7ACB2B90,
+            ROOT: 0x4B6CA198d257D755A5275648D471FE09931b764A,
+            COORDINATOR: 0xD7965D41c37B9F8691F0fe83878f6FFDbCb90996,
+            DROP: 0x44718d306a8Fa89545704Ae38B2B97c06bF11FC1,
+            MEMBERLIST: 0x6e79770F8B57cAd29D29b1884563556B31E792b0,
+            MGR: 0x5b702e1fEF3F556cbe219eE697D7f170A236cc66
+        }));
+
```

Comment:
So, just confirming--nothing for RWA006 yet, right?

Reply:
Yeah just checked, unfortunately they definitely won't be ready by tomorrow on the legal side. So it'll have to go in with a 0M debt ceiling.